be/src/column/type_column_traits.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include "column/type_traits.h"
+
+namespace starrocks {
+namespace vectorized {
+// NOTE(zc): now CppColumnTraits is only used for this class, so I move it here.
+// Someday if it is used by others, please move it into a single file.
+// CppColumnTraits
+// Infer ColumnType from FieldType
+
+template <FieldType ftype>
+struct CppColumnTraits {
```

Comment:
Looks like you could merge this traits with `ColumnTraits`, `RunTimeTypeTraits`, cause they share most of code.